connect/runtime/src/test/java/org/apache/kafka/connect/integration/SourceConnectorsIntegrationTest.java

```diff
@@ -160,8 +156,6 @@ public void testSwitchingToTopicCreationEnabled() throws InterruptedException {
         connect.assertions().assertTopicSettings(BAR_TOPIC, DEFAULT_REPLICATION_FACTOR,
                 DEFAULT_PARTITIONS, "Topic " + BAR_TOPIC + " does not have the expected settings");
 
-        connect.assertions().assertAtLeastNumWorkersAreUp(NUM_WORKERS, "Initial group of workers did not start in time.");
-
         Map<String, String> barProps = defaultSourceConnectorProps(BAR_TOPIC);
```

Comment:
nit: there's an inaccurate assertAtLeastNumWorkersAreUp string later in this test after the cluster is rolled.

Reply:
Great catch, fixed!